src/NSwag.CodeGeneration.CSharp/Templates/ClientTemplate.tt

```diff
@@ -86,7 +86,7 @@ public partial interface I<#=Model.Class#>
     }
 
 <#}#>
-    partial void PrepareRequest(System.Net.Http.HttpClient request, ref string url);
+    partial void PrepareRequest(System.Net.Http.HttpClient request, System.Text.StringBuilder urlBuilder);

```

Comment:
The only breaking change I saw was on `PrepareRequest` but, it the user needs to make changes to the url, a `StringBuilder` is better than a `string`.

There's nothing wrong with having:

    var url_ = rulBuilder_.ToString();
    ...
    request_.RequestUri = new System.Uri(url_, System.UriKind.RelativeOrAbsolute);

The compiler/runtime will, most like it, optimize it away. But, what's the point of that?

I choose to keep the breaking change as I've seen others (like `WrapSuccessResponses` to  `WrapResponses`)